src/scripts/mappings_unmapped_tables.py

```diff
@@ -0,0 +1,78 @@
+"""Generates a report of mapping progress for a given ontology.
+
+# Resources
+- GitHub issue: https://github.com/mapping-commons/disease-mappings/issues/12
+- GitHub PR: https://github.com/monarch-initiative/mondo-ingest/pull/111/
+
+TODO's:
+ - docs:
+  - add docs jinja path and output path as param? or separate scirpt?
+"""
+import os
+from argparse import ArgumentParser
+from typing import Dict, Union
+
+import pandas as pd
+
+
+# Vars
+SCRIPTS_DIR = os.path.dirname(os.path.realpath(__file__))
+PROJECT_DIR = os.path.join(SCRIPTS_DIR, '..', '..')
+ONTOLOGY_DIR = os.path.join(PROJECT_DIR, 'src', 'ontology')
+REPORTS_DIR = os.path.join(ONTOLOGY_DIR, 'reports')
+
+
+# Functions
+def run(
+    exclusions_path: str, mirror_signature_path: str, sssom_map_path: str, db_path: str, outpath_terms: str,
+    outpath_stats: str
+) -> Union[Dict[str, pd.DataFrame], None]:
+    """Run"""
+    exclusions_df = pd.read_csv(exclusions_path, header=None, names=['id']).fillna('')
+    mappable_df = pd.read_csv(mirror_signature_path, sep='\t').fillna('')
+    mappings_df = pd.read_csv(sssom_map_path, sep='\t', comment='#').fillna('')
+
+    # TODO: basic algo
+    #  - get all list of terms from mappable_df
+    #  - filter out from in exclusions_df & already in mappings_df
+
+    # TODO: get labels from .db. oak subprocess might be easier, but could try SqlImplementation
+
+    # TODO: create these
+    df_results = pd.DataFrame()
+    df_stats = pd.DataFrame()
+
+    # Sort
+    if len(df_results) > 0:
+        # TODO: what col names?
+        df_results = df_results.sort_values(['colA', '...', 'colZ'])
+
+    # Save & return
+    df_results.to_csv(outpath_terms, sep='\t', index=False)
+    df_stats.to_csv(outpath_stats, sep='\t', index=False)
+    return {outpath_terms: df_results, outpath_stats: df_results}
+
+
+def cli() -> Dict[str, pd.DataFrame]:
```

Comment:
Initial script to create the unmapped tables. I created a second script to update the documentation. The reason I separated these is because I think it might be best for there to be a single python script (and corresponding make goal) to create tables for a single ontology at a time, per our usual pattern.